client/src/ingredients/link-list.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "@reach/router";
+
+function RenderLink({ link }) {
+  return (
+    <React.Fragment key={link.uri}>
```

Comment:
I know it's my fault for bringing up the rather noisy discussion about naming it `mdn_url` or `uri` but this component uses both :)

By the way, I don't think you need to put things into sub-functions when they're only ever used for one thing. I would just take the bulk of this and put it inside the `{link.map() ...` loop on line 22-23.